packages/webapp/src/components/BarChart/Presentation.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Bar from './Bar';
+import { BarChartContainer } from './styled';
+
+const Markup = ({ items }) => (
+  <BarChartContainer>
+    {items.map((props, index) => (
+      <Bar key={items[index].amount} {...{ index, items, ...props }} />
```

Comment:
this is scary - we're assuming amounts will be unique? what happens if they're not?